rediscluster/utils.py

```diff
@@ -126,7 +126,7 @@ def parse_cluster_slots(resp, **options):
     current_host = options.get('current_host', '')
 
     def fix_server(*args):
-        return (args[0] or current_host, args[1])
+        return (nativestr(args[0]) or current_host, args[1])
```

Comment:
I would love to see a testcase in this method `test_parse_cluster_slots` that tests the same data structure but with the data format that you get back from redis when using `parse_responses=False` and without this fix. It would show that both dataformats would work as expected.